packages/i18n-utils/src/localize-url.tsx

```diff
@@ -49,22 +49,59 @@ const setLocalizedWpComPath = (
 	return url;
 };
 
-const prefixLocalizedUrlPath = (
+type PrefixOrSuffix = 'prefix' | 'suffix';
+
+const prefixOrSuffixLocalizedUrlPath = (
 	validLocales: Locale[] = [],
-	limitPathMatch: RegExp | null = null
+	limitPathMatch: RegExp | null = null,
+	prefixOrSuffix: PrefixOrSuffix
```

Comment:
Would it make sense to use a boolean flag here instead? E.g. `asPrefix = true`.

Reply:
I did actually consider using a boolean here 🙂  It’s a simpler type and just as exhaustive as the two-value union I made. 

I personally liked using a string, because I felt it made the code a bit more explicit and easier to understand — when calling the function (`prefixOrSuffixLocalizedUrlPath( x, y, 'prefix' )` vs. `prefixOrSuffixLocalizedUrlPath( x, y, true )`), but also inside the function definition itself (hence the explicit `else if` instead of just an `else`).

On the other hand, this is only used inside this file and the function is short enough, so a boolean would be a good choice too. Do you think the bool would be a better choice?

Reply:
> Do you think the bool would be a better choice?

I wouldn't say it's a better choice, but more like a personal preference. Consider it as an option suggestion, no need to change anything :)

As you pointed out, it's a private function used only within the module, so the approach shouldn't matter that much.